sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamExecution.scala

```diff
@@ -422,6 +450,12 @@ class StreamExecution(
     val replacementMap = AttributeMap(replacements)
     val triggerLogicalPlan = withNewSources transformAllExpressions {
       case a: Attribute if replacementMap.contains(a) => replacementMap(a)
+      case ct: CurrentTimestamp =>
+        CurrentBatchTimestamp(streamExecutionMetadata.currentBatchTimestampMillis,
+          ct.dataType)
+      case cd: CurrentDate =>
+        CurrentBatchTimestamp(streamExecutionMetadata.currentBatchTimestampMillis,
```

Comment:
NVM!!!! I MISSED THE conversion inside `CurrentBatchTimeStamp.toLiteral`. MY BAD!!

But then there are not tests for current_date! So one of the branch in the above code is not being tested at all.